tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/PublishToolsetBinaries.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This is mostly a duplicate of PublishCoreSetupBinaries with a
+// different regex (because version numbers are formatted differently
+// between core-setup and toolset).
+// https://github.com/dotnet/source-build/issues/554 tracks figuring
+// out a better process for doing this.
```

Comment:
Did you consider adding a bool to pick the regex instead of copying the file?

Reply:
I thought this approach would stick out more as a bad thing where a boolean might get overlooked but I'm fine with that approach if you think it's better.

Reply:
Well, to keep the task name different, could subclass and override. Or use a descriptive required string param to keep things distinguished. The thing I'm not a fan of here is that you have to do a diff to see what's different, and it just seems unnecessary in this case.